touchforms/formplayer/tests/__init__.py

```diff
@@ -1,6 +1,5 @@
 import logging
 try:
-    from touchforms.formplayer.tests.test_basic import *
```

Comment:
yes - this test was removed https://github.com/dimagi/touchforms/pull/204/files#diff-bc418358abbc497468ff998dfc0d15d0L6 because it wasn't useful anymore